sw/legacy/common/uart.c

```diff
@@ -10,6 +10,7 @@
 #define BAUD_RATE (921600)
 
 void uart_enable_rx_int(void) {
+  // TODO this must now instead be done using the PLIC
```

Comment:
Code cleanup here isn't relevant to the rest of the commit

Reply:
Moved this to a cleanup commit.